sklearn/tests/test_cross_validation.py

```diff
@@ -95,7 +95,7 @@ def fit(self, X, Y=None, sample_weight=None, class_prior=None,
     def predict(self, T):
         if self.allow_nd:
             T = T.reshape(len(T), -1)
-        return T.shape[0]
+        return T[:, 0]
```

Comment:
That seems like a big change. The semantics seem quite different. The semantics of the previous implementation of predict seem buggy to me. Could someone confirm my interpretation?